### PR TITLE
Doc fix for #1700

### PR DESCRIPTION
**Pull request by ben-arnao**

### CODE REVIEW

File: tensorflow_addons/optimizers/weight_decay_optimizers.py [hunk 1]
```diff
@@ -39,10 +39,11 @@ class DecoupledWeightDecayExtension:
 
     This class alone is not an optimizer but rather extends existing
     optimizers with decoupled weight decay. We explicitly define the two
-    examples used in the above paper (SGDW and AdamW), but in general this
-    can extend any OptimizerX by using
-    `extend_with_decoupled_weight_decay(
-        OptimizerX, weight_decay=weight_decay)`.
+    examples used in the above paper (SGDW and AdamW), but in general this can 
+    extend any OptimizerX class by using 
+        'OptimizerX = extend_with_decoupled_weight_decay(OptimizerX)'.
```
Comment (WindQAQ) [inline tensorflow_addons/optimizers/weight_decay_optimizers.py:44]:
```suggestion
        `ExtendedCls = extend_with_decoupled_weight_decay(OptimizerX)`.
```

File: tensorflow_addons/optimizers/weight_decay_optimizers.py [hunk 2]
```diff
@@ -39,10 +39,11 @@ class DecoupledWeightDecayExtension:
...
     optimizers with decoupled weight decay. We explicitly define the two
-    examples used in the above paper (SGDW and AdamW), but in general this
-    can extend any OptimizerX by using
-    `extend_with_decoupled_weight_decay(
-        OptimizerX, weight_decay=weight_decay)`.
+    examples used in the above paper (SGDW and AdamW), but in general this can 
+    extend any OptimizerX class by using 
+        'OptimizerX = extend_with_decoupled_weight_decay(OptimizerX)'. 
+    Weight decay can then be set when we initialize the optimizer: 
+        'optimizerX = OptimizerX(weight_decay=0.001, learning_rate=0.001)'.
```
Comment (WindQAQ) [inline tensorflow_addons/optimizers/weight_decay_optimizers.py:46]:
```suggestion
    Weight decay can then be set when instantiating the optimizer:
        `optimizerX = ExtendedCls(weight_decay=0.001, learning_rate=0.001)`.
```

File: tensorflow_addons/optimizers/weight_decay_optimizers.py [hunk 3]
```diff
@@ -39,10 +39,11 @@ class DecoupledWeightDecayExtension:
 
     This class alone is not an optimizer but rather extends existing
     optimizers with decoupled weight decay. We explicitly define the two
-    examples used in the above paper (SGDW and AdamW), but in general this
-    can extend any OptimizerX by using
-    `extend_with_decoupled_weight_decay(
-        OptimizerX, weight_decay=weight_decay)`.
+    examples used in the above paper (SGDW and AdamW), but in general this can
```
Comment (ashutosh1919) [inline tensorflow_addons/optimizers/weight_decay_optimizers.py:42]: Please do flake8 and black before adding commits as below.
`python -m flake8 <file_path>` to check coding style errors.
`python -m black <file_path>` to reformat the files if needed.

Comment (seanpmorgan) [reply tensorflow_addons/optimizers/weight_decay_optimizers.py:42]: As a convenience we have a git pre-commit hook you can run to validate formatting:
https://github.com/tensorflow/addons/blob/master/CONTRIBUTING.md#coding-style

**Comment by ben-arnao** [comment]
changes have been made based on feedback

**Comment by WindQAQ** [comment]
@seanpmorgan Sean, do you know what's going on with CLA? `cla:yes` is being labeled but `cla/google` status is still pending. Thank you!

**Comment by seanpmorgan** [comment]
> @seanpmorgan Sean, do you know what's going on with CLA? `cla:yes` is being labeled but `cla/google` status is still pending. Thank you!

Not a clue. Something must have hung, but CLA: Yes was already labeled so merging.

**Pull request merged**